src/middleware/oauth/github-mw.js

```diff
@@ -0,0 +1,34 @@
+const superagent = require('superagent');
+
+let getUserData = async request => {};
```

Comment:
This function on line 3 is doing nothing.  You can delete it because your getUserDatag is what is holding all the code.  Or you can change the name back to getUserData.